template-visual/main.js

```diff
@@ -3,14 +3,16 @@
 Highcharts.chart('chart-div', {
     // Setting default colors
     colors: [
-        '#C76363',
-        '#C04A49',
-        '#A42323',
-        '#7F1416',
-        '#571612',
-        '#3D3D3D',
-        '#392F83',
-        '#184F26'
+        '#800000',
+        '#FFA319',
+        '#C16622',
+        '#8F3931',
+        '#8A9045',
+        '#58593F',
+        '#155F83',
+        '#350E20',
+        '#47B5FF',
+        '#FF3399'
```

Comment:
Colors were taken from the UChi template: [Link](https://news.uchicago.edu/sites/default/files/attachments/_uchicago.identity.guidelines.pdf)